pkg/limayaml/containerd.yaml

```diff
@@ -2,8 +2,10 @@ archives:
 - location: https://github.com/containerd/nerdctl/releases/download/v1.7.6/nerdctl-full-1.7.6-linux-amd64.tar.gz
   arch: x86_64
   digest: sha256:2c841e097fcfb5a1760bd354b3778cb695b44cd01f9f271c17507dc4a0b25606
+  cid: bafybeieipdaxd3fzy3j7syzzxdaqxramk65j7ajzcqgmi6b5jyq4jgbwue
```

Comment:
I guess we don't want to use ipfs by default

Reply:
Maybe add it to the global configuration?

Reply:
Instead of just `LookPath("ipfs")` et al

Reply:
But the Content-ID (CID) needs to be inline

Reply:
Added a boolean `ipfs` toggle, off by default.